src/addon-manager.js

```diff
@@ -899,6 +900,12 @@ class AddonManager extends EventEmitter {
       // keep going
     }
 
+    if (wait) {
+      // If wait was set, wait 3 seconds + a little for the process to die.
+      // 3 seconds, because that's what is used in unloadAddon().
+      await new Promise((resolve) => setTimeout(resolve, 3500));
```

Comment:
nit: Lets make 3 seconds be a constant and use the constant + 500 for the timeout here.